lib/Twig/Extensions/Extension/Intl.php

```diff
@@ -55,12 +55,17 @@ function twig_localized_date_filter(Twig_Environment $env, $date, $dateFormat =
         'full' => IntlDateFormatter::FULL,
     );
 
+    $calendarValues = array(
+        'gregorian' => IntlDateFormatter::GREGORIAN,
+        'traditional' => IntlDateFormatter::TRADITIONAL,
```

Comment:
As in php doc:

>  The following int constants are used to specify the calendar. These calendars are all based directly on the Gregorian calendar. Non-Gregorian calendars need to be specified in locale. Examples might include locale="hi@calendar=BUDDHIST".
> 
> IntlDateFormatter::TRADITIONAL (integer)
>     Non-Gregorian Calendar 
> IntlDateFormatter::GREGORIAN (integer)
>     Gregorian Calendar 

[php doc for IntlDateFormatter](http://php.net/manual/en/class.intldateformatter.php)